### ISSUE TITLE
Show chat topics instead chat IDs in groupchats

### ISSUE THREAD

**Issue by Q-Master** [EionRobb/skype4pidgin#523]
Is it possible to show chat topics (like in web skype or orig skype) instead of chat IDs. They're unreadable.

**Comment by typoworx-de** [comment]
I agree. Makes it really hard to work with Chats if there are multiple chats used.

**Comment by Ri0n** [comment]
Works like a charm via XMPP spectrum transport though :-)
